apps/researcher/src/app/[locale]/(objects)/objects/[id]/page.tsx

```diff
@@ -112,171 +112,178 @@ export default async function Details({params}: Props) {
 
   return (
     <>
-      <div className="px-4 sm:px-10 flex gap-2 flex-row sm:justify-between">
-        <div>
-          <ToFilteredListButton className="flex items-center gap-1">
-            <ChevronLeftIcon className="w-4 h-4 fill-neutral-500" />
-            {t('backButton')}
-          </ToFilteredListButton>
-        </div>
-        <div className="sm:flex justify-end gap-4 hidden">
-          <SignedIn>
-            <ObjectListsMenu objectId={id} />
-          </SignedIn>
+      <div className="bg-consortiumBlue-800 text-white flex flex-col gap-8 pb-40">
+        <div className="px-4 sm:px-10 flex gap-2 flex-row sm:justify-between">
+          <div>
+            <ToFilteredListButton className="flex items-center gap-1">
+              <ChevronLeftIcon className="w-4 h-4 fill-white" />
+              {t('backButton')}
+            </ToFilteredListButton>
+          </div>
+          <div className="sm:flex justify-end gap-4 hidden">
+            <SignedIn>
+              <ObjectListsMenu objectId={id} />
+            </SignedIn>
+          </div>
         </div>
-      </div>
 
-      <div className="px-4 sm:px-10 my-4 flex flex-col gap-4 w-full bg max-w-[1800px] mx-auto">
-        <h1
-          className="text-2xl md:text-4xl md:items-center"
-          data-testid="page-title"
-        >
-          {object.name}
-        </h1>
+        <div className="px-4 sm:px-10 my-4 flex flex-col gap-4 w-full bg max-w-[1800px] mx-auto">
```

Comment:
The title here has the classname `max-w-[1800px]`. But this class is missing at other divs. So, on big screens, only this div has a max width, which looks weird.

![image](https://github.com/colonial-heritage/colonial-collections/assets/1481602/96d2c171-8971-424c-96e6-ade08607e427)